Admin/AdminHelper.php

```diff
@@ -335,6 +334,8 @@ private function pathExists(PropertyAccessorInterface $propertyAccessor, $entity
             return true;
         } catch (NoSuchPropertyException $e) {
             return false;
+        } catch (UnexpectedTypeException $e) {
```

Comment:
Should I do it on the master branch? This can only be removed if sf2.3 support is dropped. @OskarStark